proto/frequenz/api/common/v1/microgrid/components/components.proto

```diff
@@ -141,8 +142,29 @@ message Component {
 
   // The operational lifetime of the component.
   frequenz.api.common.v1.microgrid.Lifetime operational_lifetime = 9;
+
+  // List of rated bounds present for the component identified by Metric.
+  repeated ComponentRatedBound rated_bounds = 10;
 }
 
+// Defines a components rated bound for a specific metric.
+message ComponentRatedBound {
```

Comment:
I'd rather call this `MetricRatedBounds` since these are rated bounds for a specific metric.

Reply:
Agreed, updated :)